check/checkvscode.go

```diff
@@ -7,8 +7,11 @@ package check
 import (
 	"bufio"
 	"os/exec"
+	"strings"
 )
 
+const GoExtensionName = "ms-vscode.go"
```

Comment:
No need to export this.

const goExtensionName

Reply:
I'll change it.